src/test/java/uk/gov/hmcts/reform/professionalapi/service/LegacyPbaAccountServiceImplTest.java

```diff
@@ -0,0 +1,130 @@
+package uk.gov.hmcts.reform.professionalapi.service;
+
+import static org.assertj.core.api.Java6Assertions.assertThat;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.times;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import uk.gov.hmcts.reform.professionalapi.domain.ContactInformation;
+import uk.gov.hmcts.reform.professionalapi.domain.Organisation;
+import uk.gov.hmcts.reform.professionalapi.domain.OrganisationStatus;
+import uk.gov.hmcts.reform.professionalapi.domain.PaymentAccount;
+import uk.gov.hmcts.reform.professionalapi.domain.ProfessionalUser;
+import uk.gov.hmcts.reform.professionalapi.domain.ProfessionalUserStatus;
+import uk.gov.hmcts.reform.professionalapi.domain.UserAccountMap;
+import uk.gov.hmcts.reform.professionalapi.persistence.OrganisationRepository;
+import uk.gov.hmcts.reform.professionalapi.persistence.PrdEnumRepository;
+import uk.gov.hmcts.reform.professionalapi.persistence.ProfessionalUserRepository;
+import uk.gov.hmcts.reform.professionalapi.persistence.UserAttributeRepository;
+import uk.gov.hmcts.reform.professionalapi.service.impl.LegacyPbaAccountServiceImpl;
+import uk.gov.hmcts.reform.professionalapi.service.impl.ProfessionalUserServiceImpl;
+import uk.gov.hmcts.reform.professionalapi.service.impl.UserAttributeServiceImpl;
+
+
+public class LegacyPbaAccountServiceImplTest {
+
+
+    private final Organisation organisationMock = mock(Organisation.class);
+    private final PaymentAccount paymentAccountMock = mock(PaymentAccount.class);
+    private final UserAccountMap userAccountMapMock = mock(UserAccountMap.class);
+    private List<ProfessionalUser> usersNonEmptyList = new ArrayList<ProfessionalUser>();
+    private List<PaymentAccount> paymentAccounts = new ArrayList<PaymentAccount>();
+    private List<UserAccountMap> userAccountMaps = new ArrayList<>();
+
+
+    List<String> paymentAccountPbaNumbers = new ArrayList<>();
+    private final ProfessionalUserRepository professionalUserRepository = mock(ProfessionalUserRepository.class);
+    private final LegacyPbaAccountService legacyPbaAccountServiceImpl = mock(LegacyPbaAccountServiceImpl.class);
+
+    private final OrganisationRepository organisationRepository = mock(OrganisationRepository.class);
+
+    private final ProfessionalUser professionalUser = mock(ProfessionalUser.class);
```

Comment:
Please rename going forward so that all mocks end with the word Mock (e.g. professionalUserMock). Can be addressed in a follow up Jira.